types/bootstrap/js/dist/scrollspy.d.ts

```diff
@@ -63,6 +63,11 @@ declare namespace ScrollSpy {
          * Specifies element to apply Scrollspy plugin.
          */
         target: string | Element | JQuery;
+
+        /**
+         * `IntersectionObserver` [threshold](https://developer.mozilla.org/en-US/docs/Web/API/IntersectionObserver/IntersectionObserver#threshold) valid input, when calculating scroll position.
+         */
+        threshold: number[] | undefined;
```

Comment:
`threshold` can be `string` as well and I prefer `?` instead of `undefined`
https://github.com/twbs/bootstrap/pull/36750/files#diff-1d84c75bd9722a89a7db565ca8061c7b2c7657ed908a9cb2a2feb1124537429bR118
```suggestion
        threshold?: number[] | string;
```